diff --git a/Package.swift b/Package.swift
index f94fd8a34123fc53ba0629776919801f0f96d662..e9aa50b4590b9a5d3ad4eb42979e485d7787533f 100644
--- a/Package.swift
+++ b/Package.swift
@@ -17,8 +17,7 @@ let package = Package(
         .target(
             name: "StatusBarPlugin",
             dependencies: [
-                .product(name: "Capacitor", package: "capacitor-swift-pm"),
-                .product(name: "Cordova", package: "capacitor-swift-pm")],
+                .product(name: "Capacitor", package: "capacitor-swift-pm")],
             path: "ios/Sources/StatusBarPlugin"
         ),
         .testTarget(
