diff --git a/Package.swift b/Package.swift
index fc50a07fec384d3e4fb7386889bb0238db38d0a9..bbf749921d1c8597837ef41df112936f754c0e20 100644
--- a/Package.swift
+++ b/Package.swift
@@ -16,8 +16,7 @@ let package = Package(
         .target(
             name: "FilePickerPlugin",
             dependencies: [
-                .product(name: "Capacitor", package: "capacitor-swift-pm"),
-                .product(name: "Cordova", package: "capacitor-swift-pm")
+                .product(name: "Capacitor", package: "capacitor-swift-pm")
             ],
             path: "ios/Plugin"),
         .testTarget(
